build.gradle

```diff
@@ -34,37 +34,37 @@ repositories {
 	maven {
 		url 'https://repo.destroystokyo.com/repository/maven-public/'
 	}
-	
+
 	maven {
 		url 'http://ci.emc.gs/nexus/content/groups/aikar/'
 	}
-	
+
 	maven {
 		url 'http://nexus.hc.to/content/repositories/pub_releases'
 	}
-	
+
 //	maven {
 //		url 'http://maven.elmakers.com/repository/'
 //	}
-	
-	
+
+
 	maven {
 		url 'https://jitpack.io'
 	}
 }
 
 dependencies {
-	compile 'com.destroystokyo.paper:paper-api:1.12.1-R0.1-SNAPSHOT'
-	compile 'org.eclipse.jdt:org.eclipse.jdt.annotation:1.1.0'
-	compile 'com.google.code.findbugs:findbugs:2.0.3'
-	compile 'com.sk89q:worldguard:6.1.1-SNAPSHOT'
-	compile 'net.milkbowl.vault:Vault:1.6.6'
-	compile 'com.github.marcelo-mason:PreciousStones:24e3d4bf67b7240ae36b32be10e99d4091938c5c'
+	implementation 'com.destroystokyo.paper:paper-api:1.12.1-R0.1-SNAPSHOT'
+    implementation 'org.eclipse.jdt:org.eclipse.jdt.annotation:1.1.0'
```

Comment:
Please keep the indentation concise with the Skript standards (tabs instead of spaces).

Reply:
What do you mean use tabs or spaces? (the IDE was said; do like this to me)

Reply:
I think i did, please look again.